.github/workflows/.windows.yml

```
@@ -271,6 +271,9 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: ${{ env.GOPATH }}/src/github.com/docker/docker
+      -
+        name: Set up tracing
+        uses: ./.github/actions/setup-tracing
```

related to my previous comment, can you try:

```suggestion
        uses: ./.github/actions/setup-tracing
        working-directory: ${{ github.workspace }}
```